plugins/optimization-detective/class-od-url-metric-group.php

```diff
@@ -24,13 +24,17 @@ final class OD_URL_Metric_Group implements IteratorAggregate, Countable, JsonSer
 	/**
 	 * URL Metrics.
 	 *
+	 * @since 0.1.0
```

Comment:
not 'n.e.x.t'?

Reply:
No because this was introduced in the first release. I had forgotten to add this.